bitnami/redis/templates/redis-node-statefulset.yaml

```
@@ -375,7 +375,38 @@ spec:
             - name: metrics
               containerPort: 9121
           resources: {{- toYaml .Values.metrics.resources | nindent 12 }}
-  {{- end }}
+        {{- end }}
+        {{- if .Values.sentinel.metrics.enabled }}
+        - name: sentinel-metrics
+          image: {{ template "sentinel.metrics.image" . }}
+          imagePullPolicy: {{ .Values.sentinel.metrics.image.pullPolicy | quote }}
+          {{- if .Values.containerSecurityContext.enabled }}
+          securityContext: {{- omit .Values.containerSecurityContext "enabled" | toYaml | nindent 12 }}
+          {{- end }}
+          command:
+            - redis_sentinel_exporter{{- range $key, $value := .Values.sentinel.metrics.extraArgs }} --{{ $key }}={{ $value }}{{- end }}
+          env:
+            {{- if and .Values.sentinel.usePassword (and .Values.usePassword (not .Values.usePasswordFile)) }}
+            - name: SENTINEL_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: {{ template "redis.secretName" . }}
+                  key: {{ template "redis.secretPasswordKey" . }}
+            {{- end }}
+            {{- if and .Values.sentinel.usePassword (and .Values.usePassword .Values.usePasswordFile) }}
+            - name: SENTINEL_PASSWORD_FILE
+              value: /secrets/redis-password
+            {{- end }}
+          volumeMounts:
+            {{- if and .Values.sentinel.usePassword (and .Values.usePassword .Values.usePasswordFile) }}
```

I think the `and ` can receive 3 params directly without adding a new one this this:
```suggestion
            {{- if and .Values.sentinel.usePassword .Values.usePassword .Values.usePasswordFile }}
```